packages/google_maps_flutter/google_maps_flutter_ios/example/ios13/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -428,6 +498,59 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		68C9D7FC2AD0A84100DF9D79 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
```

Comment:
Most of these should be removable; see the list in https://github.com/flutter/flutter/wiki/Plugin-Tests#enabling-xctests-or-xcuitests